public/app/features/alerting/unified/components/rule-editor/QueryRows.tsx

```diff
@@ -142,59 +143,61 @@ export class QueryRows extends PureComponent<Props> {
           {(provided) => {
             return (
               <div ref={provided.innerRef} {...provided.droppableProps}>
-                {queries.map((query, index) => {
-                  const data: PanelData = this.props.data?.[query.refId] ?? {
-                    series: [],
-                    state: LoadingState.NotStarted,
-                  };
-                  const dsSettings = this.getDataSourceSettings(query);
+                <Stack direction="column">
```

Comment:
This diff is confusing, I just wrapped each `query + vizualization` with a `Stack`